EBMLSchema.xsd

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" version="01">
+  <xsd:element name="EBMLSchema" type="EBMLSchemaType"/>
+  <xsd:complexType name="EBMLSchemaType">
+    <xsd:sequence>
+      <xsd:element name="element" type="elementType" minOccurs="0" maxOccurs="unbounded"/>
+    </xsd:sequence>
+    <xsd:attribute name="docType" use="required"/>
+    <xsd:attribute name="version" use="required"/>
+  </xsd:complexType>
+  <xsd:complexType name="elementType">
+    <xsd:sequence>
+      <xsd:element name="documentation" type="documentationType" minOccurs="0" maxOccurs="unbounded"/>
+    </xsd:sequence>
+    <xsd:attribute name="name" use="required"/>
+    <xsd:attribute name="path" use="required"/>
+    <xsd:attribute name="id" use="required"/>
+    <xsd:attribute name="minOccurs" default="0"/>
+    <xsd:attribute name="maxOccurs" default="1"/>
```

Comment:
With this [PR](https://github.com/Matroska-Org/ebml-specification/pull/124) the default for `maxOccurs` is like `EBMLMaxOccurrence` it's infinite.